packages/engine/Source/Scene/Scene.js

```diff
@@ -3683,7 +3697,10 @@ Scene.prototype.initializeFrame = function () {
 
   this._tweens.update();
 
-  this._globeHeight = getGlobeHeight(this);
+  if (this._globeHeightDirty) {
```

Comment:
sanity check. are camera move and tile loading the only things that should make this dirty? are there any other instances of models loading or moving or anything that should affect this?

Reply:
I don't believe so– This is for the case where camera is being kept above terrain and 3D Tiles. In that case, either the camera moves to be above a new tile, or more data loads underneath the camera.